subsys/bluetooth/controller/ll_sw/ll_addr.c

```diff
@@ -36,11 +40,17 @@ u8_t *ll_addr_get(u8_t addr_type, u8_t *bdaddr)
 	return pub_addr;
 }
 
-void ll_addr_set(u8_t addr_type, u8_t const *const bdaddr)
+u32_t ll_addr_set(u8_t addr_type, u8_t const *const bdaddr)
```

Comment:
HCI status is always a one byte value. Any reason you're using u32_t instead of u8_t here?

Reply:
It was just a convention to use machine word size in return value, but I can analyse and convert all return types in a new commit.

Reply:
Ok. I'll leave this up to you. Either way is fine if you have a good justification for it. u8_t just seems a bit more intuitive to me if the value is always an HCI status.

Reply:
I say we convert all `ll_*` APIs to `u8_t` and use HCI error codes there. It makes sense in the case of the LL APIs